software/airtrail.yml

```
@@ -0,0 +1,10 @@
+name: "AirTrail"
+website_url: "https://airtrail.johan.ohly.dk"
+source_code_url: "https://github.com/johanohly/AirTrail"
+description: "A personal flight tracking system."
+licenses:
+  - GPL-3.0
+platforms:
+  - Docker
```

```suggestion
  - Docker
  - Nodejs
```

https://airtrail.johan.ohly.dk/docs/install/manual